test/solidity/Periphery/Permit2Proxy.t.sol

```diff
@@ -0,0 +1,721 @@
+// SPDX-License-Identifier: MIT
```

Comment:
When running `yarn coverage` on this branch, I get 72% line coverage for the Permit2Proxy. 
We agreed to achieve 100% line coverage for all new contracts (and solve our legacy debt piece by piece until the end of the year). 
Please make sure that this PR meets these requirements.